### PR TITLE
Fixed small issues

### PR DESCRIPTION
**Pull request by GiusyMancuso**

### CODE REVIEW

**Comment by GiusyMancuso** [comment]
[Beeradapter - onBindViewHolder](https://trello.com/c/aqXIsBOM/6-beeradapter-onbindviewholder)

**Comment by GiusyMancuso** [comment]
[BeerAdapter - viewholder](https://trello.com/c/mop818F5/5-beeradapter-viewholder)

**Comment by GiusyMancuso** [comment]
[BeerAdapter - constructor](https://trello.com/c/skX3CScS/7-beeradapter-constructor)

**Comment by GiusyMancuso** [comment]
[Remove '=' sign from app version text](https://trello.com/c/uKis4S0H/21-remove-sign-from-app-version-text)

**Comment by GiusyMancuso** [comment]
[ProfileFragment - OnCreateView](https://trello.com/c/NTarK7Ja/10-profilefragment-oncreateview)

**Review by alessiodicorato — APPROVED** [review]

File: app/src/main/java/co/develhope/chooseyourownbeer/BeerAdapter.kt [hunk 1]
```diff
@@ -1,60 +1,48 @@
...
-
-sealed class BeerAction(){
+sealed class BeerAction {
     object OnStarClick : BeerAction()
-    data class OnGoToDetailPageClick (val beer: Beer): BeerAction()
+    data class OnGoToDetailPageClick(val beer: Beer) : BeerAction()
 }
 
-class BeerAdapter(var beerList: List<Beer>, val onBeerClick: (BeerAction) -> Unit) : RecyclerView.Adapter<BeerAdapter.BeerViewHolder>() {
+class BeerAdapter(private val beerList: List<Beer>, private val onBeerClick: (BeerAction) -> Unit) :
```
Comment (Matteo-Catalano) [inline app/src/main/java/co/develhope/chooseyourownbeer/BeerAdapter.kt]: Tutto ok. Non capisco perchè qua diventi private.

**Review by Matteo-Catalano — APPROVED** [review]

**Pull request merged**